readme.rst

```diff
@@ -12,7 +12,10 @@ explore, extend, etc.)!
 The project contains the following:
 
 - A compiler, an assembler, a linker and a build system
-- Language front-ends: Brainfuck, C, c3, WebAssembly
+- Language front-ends: C, Brainfuck, WebAssembly,
+  `C3 <https://ppci.readthedocs.io/en/latest/reference/lang/c3.html>`_
+  (PPCI's own systems language, intended to address some pitfalls of C)
+- Besides well-known languages above, PPCI provides a systems programming 
```

Comment:
Should this read: `PPCI provides a systems programming C3`?

Reply:
> Should this read: PPCI provides a systems programming language C3?

Oops, sorry, that's a draft line which should be there at all. I started to write it, then figured it's out of place there, and moved further changes to docs/.../c3.rst, but missed to remove it. Pushing update now.